src/app/clients/view-client/view-client.component.html

```diff
@@ -378,7 +378,7 @@ <h3>{{clientInfo?.displayName}}</h3>
             <td mat-cell *matCellDef="let row">
               <i class="fa fa-cloud-download"></i>
               <!-- <i class="p-l-5 fa fa-eye"></i> -->
-              <i class="p-l-5 fa fa-times" (click)="$event.stopPropagation(); deleteClientDocuments(id, row.id)"></i>              
```

Comment:
There is no change on this line. Please remove this, it should not be present here.

Reply:
This line was already present here i just did the identation, should i remove this entire line?

Reply:
Keep the line, don't do the indentation.